.circleci/config.yml

```diff
@@ -297,3 +342,22 @@ workflows:
               only: /.*/
             branches:
               only: main
+
+      - sync-dags-branch:
+          name: 🔃 Update DAGs branch
+          filters:
+            tags:
+              only: /.*/
```

Comment:
do we want to do this for all tags? i thought just for main branch only

Reply:
Good point, sync'ing on tags is superfluous